.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
I've added a base `.editorconfig` file that just adds a newline always and sets Unix style line endings. What style of indentation is optimal?